tox.ini

```diff
@@ -101,7 +101,7 @@ depends =
     {coverage,functests}: tests
 commands =
     pip-sync-faster requirements/{env:TOX_ENV_NAME}.txt --pip-args '--disable-pip-version-check'
-    dev: sh bin/hypothesis --dev init
+    dev: python bin/make_db
```

Comment:
`make dev` in other apps (LMS, Via) seems to initialize the DB without a call like this one in `tox.ini` but I can't seem to figure out why that works in those apps but doesn't work in h. Is it a difference between SQLAlchemy 1.4 and 2.0? Might have to get to the bottom of it in order to apply the cookiecutter to h because the cookiecutter's `tox.ini` doesn't include this command here